kornia/color/rgb.py

```diff
@@ -77,7 +121,7 @@ def forward(self, image: torch.Tensor) -> torch.Tensor:  # type: ignore
         return bgr_to_rgb(image)
 
 
-def bgr_to_rgb(image: torch.Tensor) -> torch.Tensor:
+def bgr_to_rgb(image: torch.Tensor) -> torch.Tensor:  # type: ignore
```

Comment:
Same, as far as I know, the `# type: ignore` is only necessary in the forward call. Avoid it in the functions.

Reply:
mysterious as to why this is an unknown bug in pytorch. it leads to skipping the gradcheck in ci test build